modules/inflate_geometry.py

```diff
@@ -0,0 +1,98 @@
+"""Functions for manipulating polygon geometry"""
+
+import math
+import numpy as np
+
+
+def inflate_polygon(vertices: np.ndarray, scale_distance: int) -> np.ndarray:
+    """
+    Given a list of vertices representing a polygon geometry, offset the vertices such that the perpendicular
```

Comment:
does this handle both convex and concave geometry? could there be invalid inputs (waypoints that seem ok but isnt actually valid geeometry and will mess up the code). if so we should try and detect and return empty?

Reply:
Now that you mention it, I don't believe that it'll work with concave geometry. Some additional features will need to be added for it to work with concave.

Reply:
sounds good, you should change comment to "a convex polygon geometry"